### ISSUE TITLE
Wrong Firestore plugin instructions

### ISSUE THREAD

**Issue by gelas7** [pdfplum/pdfplum#13]
There are 2 points wrong

- Zip template url
- printBackground config field type (String -> bool)

You can start testing this extension right away.

Go to your [Firebase Storage dashboard](https://console.firebase.google.com/project/tripstop-dev/storage/tripstop-dev.appspot.com/files) in Firebase console.

Download demo.zip from [here](https://github.com/pdfplum/pdfplum/tree/main/template-samples), and upload it to Firebase Storage under **tripstop-dev.appspot.com/template**.

Go to your [Firestore dashboard](https://console.firebase.google.com/project/tripstop-dev/firestore/data/~2Finvoices) in Firebase console.

Create a document with these values (type of each field is written in parenthesis in front of it, so text=(string)... means type of text is string):

 text=(string)Lorem ipsum dolor sit amet consectetur adipisicing elit.
 flag=(string)OK
 articles=(array)
 articles[0]=(map)
 articles[0].title=(string)ABCD
 articles[0].content=(string)Abcd content
 articles[1]=(map)
 articles[1].title=(string)EFGH
 articles[1].content=(string)Efgh content
 articles[2]=(map)
 articles[2].title=(string)IJKL
 articles[2].content=(string)Ijkl content
 articles[3]=(map)
 articles[3].title=(string)MNOP
 articles[3].content=(string)Mnop content
 articles[4]=(map)
 articles[4].title=(string)QRST
 articles[4].content=(string)Qrst content
 colors=(map)
 colors.warm=(array)
 colors.warm[0]=(string)Red
 colors.warm[1]=(string)Yellow
 colors.warm[2]=(string)Orange
 colors.cold=(array)
 colors.cold[0]=(string)Green
 colors.cold[1]=(string)Blue
 colors.cold[2]=(string)Gray
 info=(map)
 info.Age=(number)38
 info.Name=(string)John Doe
 info.Birthday=(timestamp)1985/20/06
 info.Address=(string)Silicon Valley
 _pdfplum_config=(map)
 _pdfplum_config.templatePath=(string)**tripstop-dev.appspot.com/demo.zip**
 _pdfplum_config.outputFileName=(string)demo.pdf
 _pdfplum_config.chromiumPdfOptions=(map)
 _pdfplum_config.chromiumPdfOptions.printBackground=(**string**)true
 _pdfplum_config.adjustHeightToFit=(boolean)false

**Comment by sassanh** [comment]
Thanks for taking the time to report this issue. I'm willing to fix it in the `POSTINSTALL.md`.

While I see `printBackground` should become a boolean (and it is currently a string) something about the template path in the snippet you shared is confusing me:
The `TEMPLATE_PATH` parameter has been replaced with two different values one ending with `/demo.zip` the other ending with `/template`. Since the url in both of these lines:

```
Download demo.zip from [here](https://github.com/pdfplum/pdfplum/tree/main/template-samples), and upload it to Firebase Storage under tripstop-dev.appspot.com/template.
...
_pdfplum_config.templatePath=(string)tripstop-dev.appspot.com/demo.zip
```

 is coming from the same source, it confuses me how come they took two different values for you. Can you kindly double check that page and make sure what you shared above is an unmodified version of that page?

**Comment by sassanh** [comment]
So the zip template url is correct and we only need to fix the type of `printBackground` parameter, is that correct?

**Comment by sassanh** [comment]
Thanks for the clarification! I published a new version 0.14.1  in which this issue should be resolved. It takes a day or two for Firebase to review it.

It would be nice if you can double check it once it's published. Feel free to reopen this issue if needed.
